docs/pages/versions/v52.0.0/sdk/splash-screen.mdx

```diff
@@ -1,15 +1,19 @@
 ---
 title: SplashScreen
 description: A library that provides access to controlling the visibility behavior of native splash screen.
-sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-52/packages/expo-splash-screen'
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-splash-screen'
```

Comment:
```suggestion
sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-52/packages/expo-splash-screen'
```

I think this is an unwanted change 😄 . We point versioned pages to their respective SDK branch.

Reply:
😅